### PR TITLE
Bw 612

### PR DESCRIPTION
**Pull request by milansar**

### CODE REVIEW

File: projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts [hunk 1]
```diff
@@ -17,10 +17,12 @@ export const CONFIG: IConfig = {
     messagingSenderId: '704157888306'
   },
   'functionsUrl': `https://bitwiser-edu.firebaseapp.com/${appConstants.API_VERSION}`,
-  'editorUrl': 'https://bitwiser-edu.firebaseapp.com/trivia-editor',
+  'editorUrl': 'https://bitwiser-edu.firebaseapp.com/editor',
```
Comment (milansar) [inline projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts]: is Editor path correct

File: projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts [hunk 2]
```diff
@@ -17,10 +17,12 @@ export const CONFIG: IConfig = {
...
   'hightlighJsURL': 'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js',
   'hightlighCSSURL': 'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/a11y-light.min.css',
   'katexCSSURL': 'https://cdn.jsdelivr.net/npm/katex@0.10.1/dist/katex.min.css',
+  'addressByLatLongURL': 'https://maps.googleapis.com/maps/api/geocode/json',
```
Comment (milansar) [inline projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts]: remove addressByLatlong and addressSuggestion as it comes from constant and does not have place in enviornement file

File: projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts (hunk 1 above)
Comment (yugank1991) [reply projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts]: yes, editor path is correct

File: projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts (hunk 2 above)
Comment (yugank1991) [reply projects/shared-library/src/lib/environments/bitwiser-edu/environment.dev.ts]: I have removed both the variables from environment file

**Comment by milansar** [comment]
@anihalaney , PR is ready for your review

**Pull request merged**